src/interface/interface.cpp

```diff
@@ -874,298 +846,6 @@ int saveConfig(char const * const _filename)
-	{
-		fp->printf("/showgametimer\n");
-	}
-	if (disable_messages)
-	{
-		fp->printf("/disablemessages\n");
-	}
-	if (right_click_protect)
-	{
-		fp->printf("/right_click_protect\n");
-	}
-	if (auto_appraise_new_items)
-	{
-		fp->printf("/autoappraisenewitems\n");
-	}
-	if (startfloor)
-	{
-		fp->printf("/startfloor %d\n", startfloor);
-	}
-	/*if (splitscreen)
-	{
-		fp->printf("/splitscreen\n");
-	}*/
-	if ( useModelCache )
-	{
-		fp->printf("/usemodelcache\n");
-	}
-	fp->printf("/lastcharacter %d %d %d %d\n", lastCreatedCharacterSex, lastCreatedCharacterClass, lastCreatedCharacterAppearance, lastCreatedCharacterRace);
-	fp->printf("/gamepad_deadzone %d\n", gamepad_deadzone);
-	fp->printf("/gamepad_trigger_deadzone %d\n", gamepad_trigger_deadzone);
-	fp->printf("/gamepad_leftx_sensitivity %d\n", gamepad_leftx_sensitivity);
-	fp->printf("/gamepad_lefty_sensitivity %d\n", gamepad_lefty_sensitivity);
-	fp->printf("/gamepad_rightx_sensitivity %d\n", gamepad_rightx_sensitivity);
-	fp->printf("/gamepad_righty_sensitivity %d\n", gamepad_righty_sensitivity);
-	fp->printf("/gamepad_menux_sensitivity %d\n", gamepad_menux_sensitivity);
-	fp->printf("/gamepad_menuy_sensitivity %d\n", gamepad_menuy_sensitivity);
-	if (gamepad_rightx_invert)
-	{
-		fp->printf("/gamepad_rightx_invert\n");
-	}
-	if (gamepad_righty_invert)
-	{
-		fp->printf("/gamepad_righty_invert\n");
-	}
-	if (gamepad_leftx_invert)
-	{
-		fp->printf("/gamepad_leftx_invert\n");
-	}
-	if (gamepad_lefty_invert)
-	{
-		fp->printf("/gamepad_lefty_invert\n");
-	}
-	if (gamepad_menux_invert)
-	{
-		fp->printf("/gamepad_menux_invert\n");
-	}
-	if (gamepad_menuy_invert)
-	{
-		fp->printf("/gamepad_menuy_invert\n");
-	}
-	fp->printf("/skipintro\n");
-	fp->printf("/minimaptransparencyfg %d\n", minimapTransparencyForeground);
-	fp->printf("/minimaptransparencybg %d\n", minimapTransparencyBackground);
-	fp->printf("/minimapscale %d\n", minimapScale);
-	fp->printf("/minimapobjectzoom %d\n", minimapObjectZoom);
-	if ( uiscale_charactersheet )
-	{
-		fp->printf("/uiscale_charsheet\n");
-	}
-	if ( uiscale_skillspage )
-	{
-		fp->printf("/uiscale_skillsheet\n");
-	}
-	fp->printf("/uiscale_inv %f\n", uiscale_inventory);
-	fp->printf("/uiscale_hotbar %f\n", uiscale_hotbar);
-	fp->printf("/uiscale_chatbox %f\n", uiscale_chatlog);
-	fp->printf("/uiscale_playerbars %f\n", uiscale_playerbars);
-	if ( hide_playertags )
-	{
-		fp->printf("/hideplayertags\n");
-	}
-	if ( hide_statusbar )
-	{
-		fp->printf("/hidestatusbar\n");
-	}
-	if ( show_skill_values )
-	{
-		fp->printf("/showskillvalues\n");
-	}
-	if ( disableMultithreadedSteamNetworking )
-	{
-		fp->printf("/disablenetworkmultithreading\n");
-	}
-	if ( disableFPSLimitOnNetworkMessages )
-	{
-		fp->printf("/disablenetcodefpslimit\n");
-	}
-#ifdef USE_EOS
-	if ( LobbyHandler.crossplayEnabled )
-	{
-		fp->printf("/crossplay\n");
-	}
-#endif // USE_EOS
-
-	if ( !gamemods_mountedFilepaths.empty() )
-	{
-		std::vector<std::pair<std::string, std::string>>::iterator it;
```

Comment:
Needs a new equivalent saved into the config file before we delete

Reply:
done